headers/assert_h.md

```diff
@@ -1,8 +1,63 @@
 # \<assert.h\>
 
-Reference: [\<assert.h\>](https://en.cppreference.com/w/c/assert)
+C Reference: [\<assert.h\>](https://en.cppreference.com/w/c/error)
 
-## Types
+Relevant Zig Documentation:
+  - [std.debug.assert](https://ziglang.org/documentation/master/std/#std;debug.assert)
 
-## Functions
+## [assert()](https://en.cppreference.com/w/c/error/assert)
 
+For the following C code, which will immediately abort the program if the condition is false. 
+
+```c
+double x = -1.0;
+assert(x >= 0.0);
+```
+
+**Equivalent Zig implementation:**
+
+```zig
+const assert = std.debug.assert;
+// ...
+const x: double = -1.0;
```

Comment:
there's no double in zig, instead it would be `f64` (same thing for int used elsewhere)

Reply:
oh. sorry about that. i will fix that.